lib/api/mapBenefits.js

```diff
@@ -77,11 +80,15 @@ export function MapEICard(eiBenefitData) {
         ),
         FormatSummary(
           SummaryTypes.EstimatedDecisionDate,
-          estimatedDateOfDecision.setMonth(
-            estimatedDateOfDecision.getMonth() + 1
+          estimatedDateOfDecision.setDate(
+            estimatedDateOfDecision.getDate() + 28
           )
         ),
       ]
+      //remove estimated decision date if current day is past that point
+      if (Date.now() > estimatedDateOfDecision.getTime()) {
```

Comment:
This check would be better done, before adding the date. Check for other spots where this could be improved as well.